### PR TITLE
Disabling data ranges

### PR DESCRIPTION
**Pull request by kmgv**
Added ability do defines date ranges where dates are disabled.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,7 +151,14 @@ var state = {
             new Date(2016, 9, 16),
             new Date(2016, 9, 17),
             new Date(2016, 9, 18)
-        ]
+        ],
+        ranges: [{ // Disable dates in given ranges (exclusive).
+            from: new Date(2016, 11, 25),
+            to: from: new Date(2016, 11, 30)
```
Comment (charliekassel) [inline README.md]: is this a typo?

File: src/components/Datepicker.vue [hunk 2]
```diff
@@ -581,6 +581,16 @@ export default {
       if (typeof this.disabled.from !== 'undefined' && this.disabled.from && date > this.disabled.from) {
         disabled = true
       }
+      if (typeof this.disabled.ranges !== 'undefined') {
+        this.disabled.ranges.forEach((r) => {
```
Comment (charliekassel) [inline src/components/Datepicker.vue]: Can you use a more verbose variable please? e.g `range`

File: test/unit/specs/Datepicker.spec.js [hunk 3]
```diff
@@ -331,6 +331,27 @@ describe('Datepicker disabled dates', () => {
   })
 })
 
+describe('Datepicker respects disabled ranges', () => {
```
Comment (charliekassel) [inline test/unit/specs/Datepicker.spec.js]: 👍

**Comment by kmgv** [comment]
Commited requested changes

**Pull request merged**